security/tasks.py

```diff
@@ -30,23 +30,19 @@ class LoggedTask(Task):
     abstract = True
     logger_level = logging.WARNING
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.output = StringIO()
-
-    @property
-    def stdout(self):
-        return self.output
-
-    @property
-    def stderr(self):
-        return self.output
+    def push_request(self, *args, **kwargs):
+        task_id = self.request.id
+        output_stream = self.request.output_stream
+        super().push_request(*args, **kwargs)
+        self.request.id = task_id
+        self.request.output_stream = output_stream
```

Comment:
tohle je velká sranda, dělá to jen když je zapnuto CELERY_ALWAYS_EAGER. Prostě přes spuštěním commandu to přepíše context a odstraní i task_id což dost dobře nechápu proč to je takto blbě napsané. Nicméně já si to tam prostě vrátil zpět